.github/workflows/codecov.yml

```diff
@@ -0,0 +1,28 @@
+name: Run tests and upload coverage
+
+on: 
+  push
```

Comment:
Would it not be good idea to have code coverage workflow on PR to see code coverage is decrease on increase after new changes on PR ?

Reply:
sure @ajaypratap003 adding..